test-it/karma/publicapi/FindValueRef.Test.js

```diff
@@ -0,0 +1,27 @@
+describe('The Vireo EggShell findValueRef api can', function () {
+    'use strict';
+    // Reference aliases
+    var Vireo = window.NationalInstruments.Vireo.Vireo;
+    var vireoRunner = window.testHelpers.vireoRunner;
+    var fixtures = window.testHelpers.fixtures;
+
+    var vireo = new Vireo();
+
+    var publicApiMultipleTypesViaUrl = fixtures.convertToAbsoluteFromFixturesDir('publicapi/MultipleTypes.via');
+    var viName = 'MyVI';
+
+    beforeAll(function (done) {
+        fixtures.preloadAbsoluteUrls([
+            publicApiMultipleTypesViaUrl
+        ], done);
+    });
+
+    beforeAll(function () {
+        vireoRunner.rebootAndLoadVia(vireo, publicApiMultipleTypesViaUrl);
+    });
+
+    it('find a double', function () {
```

Comment:
Maybe add another test for the error case. Either when the viName was not declared or the path doesn't exist.